### DISCUSSION TITLE
Logfiles in s3

### DISCUSSION THREAD

**Discussion by florisbosch** [opcodesio/log-viewer#173] [Q&A] [answered]
Hi,

Firstly, nice package! It looks very beautiful.

We run our Laravel backend API on Amazon ECS, using multiple containers. 
However, since they don't have shared storage, we don't have complete logfiles.

Is it possible to combine the logs and send them to S3, so that the log viewer can read them directly or stream them from S3? This would make it easier for our developers to find the logs. 
Currently, we use CloudWatch and Sentry, but I think this solution would be more convenient.

Is there a way to do this? Thank you for your help.

**Comment by arukompas** [comment, marked as answer]
Hey @Florisbosch 

Unfortunately that's not supported at the moment. Log Viewer can only read from local storage as is the case for most Laravel apps :) 

But that's a great idea though!
